lib/Runtime/Library/JavascriptObject.cpp

```diff
@@ -2123,6 +2123,8 @@ namespace Js
             scriptContext->GetLibrary()->SetArrayObjectHasUserDefinedSpecies(true);
         }
 
+        JavascriptLibrary::CheckAndInvalidateIsConcatSpreadableCache(propId, scriptContext);
```

Comment:
Is it necessary to catch this case in this method if we're already catching it in the type handler code?